lib/Dialect/AIEVec/Transforms/IntervalReuse.cpp

```diff
@@ -33,8 +33,8 @@ bool IntervalReuse::sameEnclosingLoops(
   // reference.
   Operation *ref = extentMap.begin()->first;
 
-  Block *opBlock = op->getBlock();
-  Block *refBlock = ref->getBlock();
+  [[maybe_unused]] Block *opBlock = op->getBlock();
+  [[maybe_unused]] Block *refBlock = ref->getBlock();
```

Comment:
The solution to this is not to mark them as `maybe unused` but to actually use them in the return statement.

Reply:
Yes, that's true. The reason I chose to do it like this was to avoid changing the code of the original designer too much. I'll make the change you suggested, I also prefer it.